tests/test_maf.py

```diff
@@ -58,6 +58,7 @@
     ["maf", "filter", "non_common_variant", "--help"],
     ["maf", "filter", "non_hotspot", "--help"],
     ["maf", "filter", "not_complex", "--help"],
+    ["maf", "filter", "access_remove_variants", "--help"],
```

Comment:
It sounds like we'll need to combine these steps (filter and tag) into a single command, similar to what we did for ch:
- process script: [https://github.com/msk-access/postprocessing_variant_calls/blob/203aeab7882fbf7943[…]83ee8d09989/postprocessing_variant_calls/maf/tag/tag_process.py](https://github.com/msk-access/postprocessing_variant_calls/blob/203aeab7882fbf79430d76e7526b883ee8d09989/postprocessing_variant_calls/maf/tag/tag_process.py#L201-L232)
- tagging function: [https://github.com/msk-access/postprocessing_variant_calls/blob/203aeab7882fbf7943[…]76e7526b883ee8d09989/postprocessing_variant_calls/maf/helper.py](https://github.com/msk-access/postprocessing_variant_calls/blob/203aeab7882fbf79430d76e7526b883ee8d09989/postprocessing_variant_calls/maf/helper.py#L332-L338). It re-uses